.github/workflows/schema-validation.yml

```diff
@@ -25,32 +24,16 @@ jobs:
         run: |
           microdnf install -y findutils
 
-      - name: Validate schema files.
+      - name: Linter
         run: |
-          for file in $(find schema -name "*.yml"); do
-            echo "Testing schema $file"
-            simtools-validate-file-using-schema \
-              --file_name $file \
-              --log_level DEBUG
-          done
+          pip install yamllint
+          yamllint -d "{rules: {line-length: {max: 150}}}" schema/*.yml
 
-      - name: Validate parameter files.
-        run: |
-          for file in $(find verified_model -name "*.json"); do
-            PARAM=$(basename $file .json)
-            echo "Testing parameter $file"
-            simtools-validate-file-using-schema \
-            --file_name $file \
-            --log_level DEBUG
-          done
-
-      - name: Validate metadata in parameter files.
+      - name: Validate schema files.
         run: |
-          for file in $(find verified_model -name "*.json"); do
-            PARAM=$(basename $file .json)
-            echo "Testing metadata for parameter $file"
+          for file in schema/*.yml; do
```

Comment:
Here you don't need to use the better version of the for loop as you did above?

Reply:
No, as we don't have to recursively read files in subdirectories here, this simple `for` loop is working (and accepted by shellcheck).